src/js/api.js

```diff
@@ -1,5 +1,82 @@
 class LeaderboardApi {
-  init = () => 0;
+  apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/YGUJElUwMihyneuKyz3n/scores';
+
+  constructor({ refreshBtn, submitBtn, board }) {
+    this.refreshBtn = document.getElementById(refreshBtn);
+    this.submitBtn = document.getElementById(submitBtn);
+    this.board = document.querySelector(board);
+  }
+
+  init = () => {
+    this.refreshBtn.addEventListener('click', () => this.getScores());
+    this.submitBtn.addEventListener('click', (e) => this.submitForm(e));
+    this.getScores();
+  };
+
+  newGame = () => {
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+      method: 'POST',
+      body: JSON.stringify({
+        name: 'WhidBB GameOne',
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json())
+      .then((json) => console.log(json));
```

Comment:
- [ ] According to the general Javascript requirements:

![image](https://user-images.githubusercontent.com/57421823/173753606-1147bbe4-a36b-4288-87a2-f971418e6a52.png)

Therefore, please remove this console.log method call in order to improve the cleanliness of the repo 💪